src/Docs/SharedView.fs

```diff
@@ -1,9 +1,19 @@
-﻿module Docs.SharedView
+module Docs.SharedView
 
 open Feliz
 open Router
 open Feliz.DaisyUI
 
+type Msg =
```

Comment:
Moved these to shared, so pages modify state

Reply:
I think this should stay on the top level and we should pass callback to Countdown component, but I'll change that later, no problem.

Reply:
🆗